test/unit/logic/transactions/pool.js

```diff
@@ -0,0 +1,1412 @@
+'use strict';
+
+var expect = require('chai').expect;
+var sinon  = require('sinon');
+var _ = require('lodash');
+var rewire = require('rewire');
+var slots = require('../../../../helpers/slots.js');
+
+var node = require('../../../node');
+var DBSandbox = require('../../../common/globalBefore').DBSandbox;
+var modulesLoader = require('../../../common/modulesLoader');
+var bson = require('../../../../helpers/bson.js');
+var constants = require('../../../../helpers/constants.js');
+var TransactionPool = require('../../../../logic/transactions/pool.js');
+
+describe('transactionPool', function () {
+
+	var testAccounts = [
```

Comment:
This is alot of lines for fixture data. can we put them in a seperate file maybe?

Reply:
We have issue #987 for that